tests/system/test_system.py

```diff
@@ -93,7 +93,12 @@ def _list_instances():
 
 
 def setUpModule():
-    Config.CLIENT = Client()
+    if USE_EMULATOR:
+        from google.auth.credentials import AnonymousCredentials
+        emulator_project = os.getenv("GCLOUD_PROJECT") or 'emulator-test-project'
```

Comment:
nits: I think you can do `os.getenv("GCLOUD_PROJECT", "emulator-test-project")`.